src/audits/manifest/short-name-length.js

```diff
@@ -48,16 +49,26 @@ class ManifestShortNameLength extends Audit {
   static audit(artifacts) {
     let isShortNameShortEnough = false;
     let debugString;
+    let resolvedName;
     const manifest = artifacts.manifest.value;
     const suggestedLength = 12;
 
-    if (manifest && manifest.short_name && manifest.short_name.value) {
+    if (manifest) {
+      const shortNameValue = _get(manifest, 'short_name.value');
```

Comment:
even with `_get` this is more complicated than it needs to be. [`short_name` and `name` are always defined on the manifest](https://github.com/GoogleChrome/lighthouse/blob/500ba29394133706579d0002443ee686aef26f21/src/lib/manifest-parser.js#L280-L281); it's their `value` properties that may be `undefined`. Since `manifest` is already checked above, this whole block can be as simple as `resolvedName = manifest.short_name.value || manifest.name.value;`